docs/guide/how-to/deploying-with-webpack.mdx

```diff
@@ -0,0 +1,73 @@
+import { Tabs, Tab, Steps, PackageManagerTabs } from '@theme';
+
+# Deploying with Webpack
+You can deploy an existing Webpack-bundled application to Zephyr Cloud.
```

Comment:
all instances of webpack should be fully lower case.

Reply:
Fixed in [56d6890](https://github.com/ZephyrCloudIO/zephyr-documentation/pull/5/commits/56d68905f2f54d8be2e397004478d6708e77b1c8)